PluginCode/src/main/java/smr/cs/ualberta/libcomp/utils/PositionalXMLReader.java

```diff
@@ -0,0 +1,90 @@
+package smr.cs.ualberta.libcomp.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Stack;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.xml.sax.Attributes;
+import org.xml.sax.Locator;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+public class PositionalXMLReader {
```

Comment:
Can you add comments at the top for author, and what it's for

Reply:
I don't find the code in github, but according to where I copy from, it says the method suggested by `Michael Kay`
https://github.com/ualberta-smr/LibCompPlugin/pull/54/files#diff-a24862e405ebe126e9402222bc2803c26da8fc869e9999a8c41470cb8bb0b47aR21-R26

Reply:
Ok, say that it was copied from Stack Overflow and mention the author and the Stack Overflow license "Used in accordance to Stack Overflow license [https://stackoverflow.com/help/licensing](https://stackoverflow.com/help/licensing)"